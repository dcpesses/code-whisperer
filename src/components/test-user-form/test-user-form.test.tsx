import {render} from '@testing-library/react';
import TestUserForm from './index';

describe('TestUserForm', () => {
  test('Should render as expected', () => {
    const {container} = render(<TestUserForm />);
    expect(container).toMatchSnapshot();
  });
});
