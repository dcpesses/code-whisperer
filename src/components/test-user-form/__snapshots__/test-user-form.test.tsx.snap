// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`TestUserForm > Should render as expected 1`] = `
<div>
  <div
    class="text-center"
  >
    <input
      placeholder="Enter a Streamer"
      type="text"
      value=""
    />
    <button>
      Load
    </button>
    <div
      class="pt-2"
    >
      <button
        class="btn btn-primary"
      >
        Log Out
      </button>
    </div>
  </div>
</div>
`;
